swaggregate.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+import argparse
+import logging
+import yaml
+
+LOGGER = logging.getLogger(__name__)
+
+
+def extract_includes(filepath='main.yaml'):
+    """Given the provided *filepath*, load the list of swaggregator include
+    statements, in the order that they appear in *filepath*
+
+    :param filepath: The path to the swaggregator compliant yaml file
+    :return: A :const:`list` of swagger files to include
+    """
+    includes = []
+    include_stmt = '#include:'
+    with open(filepath) as f:
+        comment_lines = (l for l in f if l.startswith('#'))
+        for line in comment_lines:
+            if line.startswith(include_stmt):
```

Comment:
Why don't you move  the `.startswith(include_stmt)` into the `comment_lines` generator's `l.startswith(include_stmt)`?